Gemfile

```diff
@@ -17,6 +18,7 @@ gem "rails-i18n"
 gem "sass-rails", ">= 6"
 gem "turbolinks", "~> 5"
 gem "webpacker", "~> 4.0"
+gem "will_paginate", "3.3.0"
```

Comment:
please use https://github.com/kaminari/kaminari
remove will_paginate
kaminari is used more and more commonly